test.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+from Part_Three import *
+
+class TestDataMunging(unittest.TestCase):
+
+
+    def test_compare_difference(self):
+        objBigger = PartThree_Shared_Functionality.SharedObject('obj1', 10, 0)
+        objSmaller = PartThree_Shared_Functionality.SharedObject('obj2', 0, 0)
+
+        result = compare_difference(objBigger, objSmaller)
+        self.assertEqual(result, objSmaller)
+
+
+    # tests for csv football data
+    def test_check_validity_of_the_object_soccerTrue(self):
+        result = PartThree_SoccerLeagueTable.check_validity_of_the_object('10', '2', 'Arsenal')
+        self.assertTrue(result)
+
+
+    def test_check_validity_of_the_object_soccerFalseChar(self):
+        result = PartThree_SoccerLeagueTable.check_validity_of_the_object('c', '2', 'Arsenal')
```

Comment:
The style guide for python limits the line length to 79 characters.
but make sure that you don't just break the line in the middle, instead re-organize your code to make it readable beyond one line by using temporary variables, helper functions, etc.